src/routes/legal/privacy-policy.md

```diff
@@ -19,7 +19,9 @@ If you want the data we've collected on you to be sent, please contact us on Mat
 
 We have Plausible analytics installed on our website (the one you are on right now!), and our [Blog](https://blog.projectsegfau.lt), which means we can just easily see what part of our site you're on, what country you are from, what platform you are from, potentially a referrer and all those get clumped up together.
 
-It is about the same as nginx logging. We use analytics in order to see how many visitors we get every day, what pages get visited the most, what countries visit our website the most etc. uBlock and other privacy extensions block Plausible by default.
+**Plausible analytics does not collect any PII(Personally Identifiable Information)**
```

Comment:
I think this would be better: 
```
Plausible Analytics is completely anonymous. We can't exactly figure out who you are through Plausible. It is also GDPR compliant.
```

Reply:
> I think this would be better:
> 
> ```
> Plausible Analytics is completely anonymous. We can't exactly figure out who you are through Plausible. It is also GDPR compliant.
> ```

`Plausible Analytics are` would be better

Reply:
??? English is confusing